### ISSUE TITLE
HELP

### ISSUE THREAD

**Issue by 6FaNcY9** [cococry/Ragnar#7]
I wanted to install the WM on pop OS but when i logout and change the desktop i just get a black screen. Also i tried to launch it with the tty i also get a black screen could u maybe tell me where to view for logs for debugging. And some common issues that i have to look for. 

```
FILE: $HOME/.xsession-errors

`Xsession: X session started for vino at Tue May 16 12:47:28 PM CEST 2023
dbus-update-activation-environment: setting DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
dbus-update-activation-environment: setting DISPLAY=:1
dbus-update-activation-environment: setting XAUTHORITY=/home/vino/.Xauthority
localuser:vino being added to access control list
dbus-update-activation-environment: setting GTK_MODULES=gail:atk-bridge
dbus-update-activation-environment: setting QT_ACCESSIBILITY=1`
```

THX :D

**Comment by cococry** [comment]
when you start the WM from tty with startx, it also blackscreens?

**Comment by 6FaNcY9** [comment]
yes maybe the config.h is missing and the content of .xinitrc is exec ragnar

**Comment by cococry** [comment]
so you cannot get past startx in any way?

and if not, please send what startx outputs after crash

**Comment by 6FaNcY9** [comment]
yes but it doesnt crash it just stays black 

maybe its becaus of my nvidia gpu

**Comment by cococry** [comment]
try installing alacritty and then press super + enter in the blackscreen. this should open alacritty

maybe its just because you dont have a wallpaper set and there is a bug that the bar is not showing so this could look like a blackscreen.

**Comment by cococry** [comment]
and it's probably not because of your nvidia gpu because i also have one (rtx 3070) and it works

**Comment by 6FaNcY9** [comment]
yes your right i can start alacritty, it just doesnt show the bar and no wallpaper

**Comment by cococry** [comment]
perfect! 

i suggest that in your .xinitrc add "nitrogen --restore &" and install nitrogen and set a wallpaper and for the bar, just fullscreen a window and unfullscreen it. this should show the bar. or just press super + i two times.

**Comment by 6FaNcY9** [comment]
boath solutions doesnt work should i put config.h in $HOME/.config/ragnar/config.h because there is no config file for the vm

**Comment by cococry** [comment]
the config is managed like in dwm. there is a config.h file in the repo and you have to edit the macro values. than recompile with "sudo make ragnar clean install".

**Comment by 6FaNcY9** [comment]
thx you very much for the quick response 😇 after school I will try it on my other maschine are there any log files that get created to show why top bar isnt launching?

**Comment by cococry** [comment]
is the bar also not showing when you hit super + i two times?

**Comment by cococry** [comment]
This issue has been addressed in the latest release v1.1 (https://github.com/cococry/Ragnar/releases/tag/1.1). After opening a window, the bar will be displayed.
